### ISSUE TITLE
Smarter @-replies

### ISSUE THREAD

**Issue by nogweii** [zigdon/twirssi#4]
I use twirssi with both my identi.ca account as well as my twitter account. Could twrissi detect who I am replying to and use that account? E.g. there is a @cloudhead on twitter, but not on Identi.ca. I have identi.ca set as my default account, as it also tweets my 'dents. But there isn't a @cloudhead on identi.ca - no matter, there was never a (seen) dent from a user by that name. Therefore, since twirssi only saw @cloudhead in the twitter account, I think it should have used twitter instead of identi.ca, for this one tweet.

TL;DR: Use the correct account for the replying tweet/dent. (As normally replies aren't propogated across accounts)
